internal/suites/example/compose/authelia/docker-compose.frontend.dist.yml

```diff
@@ -4,5 +4,15 @@ services:
     image: nginx:alpine
     volumes:
       - './example/compose/authelia/resources/nginx.conf:/etc/nginx/nginx.conf'
+    labels:
+      # Traefik 1.x
+      - 'traefik.frontend.rule=Host:login.example.com'
+      # Traefik 2.x
+      - 'traefik.http.routers.authelia_frontend.rule=Host(`login.example.com`)'
+      - 'traefik.http.routers.authelia_frontend.entrypoints=https'
+      - 'traefik.http.routers.authelia_frontend.tls=true'
+      - 'traefik.http.services.authelia-frontend-authelia.loadbalancer.server.port=3000'
```

Comment:
```suggestion
      - 'traefik.http.services.authelia_frontend.loadbalancer.server.port=3000'
```

Reply:
Actually I don’t believe this is even necessary. Traefik will pick up the port from the expose param so this shouldn’t be required.

Reply:
On second thought default nginx exposes 80 iirc so you’ll have multiple. Might need to double check if there isn’t a default expose by adding it in the compose that will suffice. If not you will need this.

Reply:
I actually added it because it was not able to pick it up correctly.